### PR TITLE
Fix minor typo

### PR DESCRIPTION
**Pull request by arturobernalg**

### CODE REVIEW

File: commons-statistics-distribution/src/test/java/org/apache/commons/statistics/distribution/BaseDiscreteDistributionTest.java [hunk 1]
```diff
@@ -65,7 +65,7 @@
  * The parameters are parsed from String values to the appropriate parameter object. Currently
  * this supports Double and Integer; numbers can be unboxed and used to create the distribution.
  *
- * <p>Illegal arguments for the distribution are tested from parametersprovided by
+ * <p>Illegal arguments for the distribution are tested from parameters-provided by
```
Comment (aherbert) [inline commons-statistics-distribution/src/test/java/org/apache/commons/statistics/distribution/BaseDiscreteDistributionTest.java:68]: No hyphen required here

File: commons-statistics-distribution/src/test/java/org/apache/commons/statistics/distribution/TruncatedNormalDistributionTest.java [hunk 2]
```diff
@@ -200,7 +200,7 @@ void testVariance() {
     /**
      * Test cases for computation of the moments. This hits edge cases including truncations
      * too extreme to have a probability range for the distribution.
-     * The test ensures that the moments are computable for parameterisations
+     * The test ensures that the moments are computable for parameters-provided
```
Comment (aherbert) [inline commons-statistics-distribution/src/test/java/org/apache/commons/statistics/distribution/TruncatedNormalDistributionTest.java:203]: Parameterisations is the intended term. It is a British alternative spelling of [parametrization](https://en.wiktionary.org/wiki/parametrization). It is intended to describe the result of parameters, i.e. 'computable for the result of parameters (the distribution under test) where the bounds fall within'. Feel free to update it to the US english which is more standard for commons components.

**Review by aherbert — CHANGES_REQUESTED** [review]
Just a few minor changes required.

File: commons-statistics-distribution/src/test/java/org/apache/commons/statistics/distribution/BaseDiscreteDistributionTest.java (hunk 1 above)
Comment (arturobernalg) [reply commons-statistics-distribution/src/test/java/org/apache/commons/statistics/distribution/BaseDiscreteDistributionTest.java]: done

File: commons-statistics-distribution/src/test/java/org/apache/commons/statistics/distribution/TruncatedNormalDistributionTest.java (hunk 2 above)
Comment (arturobernalg) [reply commons-statistics-distribution/src/test/java/org/apache/commons/statistics/distribution/TruncatedNormalDistributionTest.java]: reverted
